src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```
@@ -17,6 +17,14 @@ $avatarUrl: var(--user-profile-picture-avatar-url, none);
 
   cursor: pointer;
 
+  &__onlyIcon {
```

```suggestion
  &--only-icon {
```

and in the places it is used

---

c4879fa